src/core/address_finder.rs

```diff
@@ -113,28 +112,18 @@ mod os_impl {
         })
     }
 
-    fn get_ruby_binary(maps: &Vec<MapRange>) -> Result<Binary, Error> {
-        let map: &MapRange = maps.iter()
-            .find(|ref m| if let Some(ref pathname) = m.filename() {
-                pathname.contains("bin/ruby") && m.is_exec()
```

Comment:
I have some concerns about `m.is_exec()` — there's no such field on `DyldInfo`... not sure how important this part is.

Reply:
I don't remember if I put that there for a reason or not, so this seems fine